coalib/coala.py

```diff
@@ -61,13 +66,19 @@ def main():
 
             return 0
 
+        elif args.ci:
+            exitcode = mode_ci(args)
+            return exitcode
+
+        elif args.format:
+            exitcode = mode_format()
+            return exitcode
+
     except BaseException as exception:  # pylint: disable=broad-except
         return get_exitcode(exception, log_printer)
 
     import functools
 
-    from coalib.coala_main import run_coala
-
     partial_print_sec_beg = functools.partial(
```

Comment:
all the code in this line and following should go into another normal_mode function right?